cake/package-definitions.cake

```diff
@@ -26,6 +27,7 @@ public void InitializePackageDefinitions(ICakeContext context)
         NetCore21Test,
         NetCore31Test,
         Net50Test,
+        Net60Test,
         NetCore21PlusNetCore31PlusNet50Test
```

Comment:
Good Idea. I just changed that one to include all four. Also added a test of running .NET  Framework 4.0 and .NET 6.0 together, since we didn't have an FX / Core combination test.